mod/carrierwave/set/abstract/attachment/coded.rb

```diff
@@ -29,13 +29,17 @@ def storage_type_error error_name
 end
 
 def mod_from_content
-  if (m = content&.match %r{^:[^/]+/([^.]+)})
+  if (m = mod_name_match content) # || mod_name_match(db_content_before_act))
```

Comment:
The remaining failing spec is connected to the `#write_temporary_identifier` method in mounter.rb in carrierwave. (not a typo; mounter not mount). I tried a hacky workaround, but it broke stuff.